tools/vf-component-library/src/site/_includes/layouts/components.njk

```diff
@@ -372,19 +372,21 @@ import { {{ component.baseHandle | camelize(true) | replace("-", "") }} } from "
         {{ readmeInstallation | replace("## Install", "") | markdown | safe }}
       </details>
       {% endif %}
-
-      {% if 'CHANGELOG.md' | length %}
-      {% markdown %}
-      <div class="vf-stack vf-stack--custom" style="--vf-stack-margin--custom: .5rem;">
-      <h3>Changelog</h3>
-      <details class="vf-details" id="changelog">
-        <summary class="vf-details--summary">Changelog</summary>
-
-        {% include 'CHANGELOG.md' ignore missing %}
-      </details>
-      </div>
-      {% endmarkdown %}
-      {% endif %}
+      {# Iterate through the array of assets for current component and check for CHANGELOG.md. Include it if found. #}
+      {%- for asset in component.resources.assets %}
+        {% if asset.name === 'CHANGELOG.md' %}
```

Comment:
can we use `component.baseHandle + '/CHANGELOG.md'` instead of loop

Reply:
Thanks, updated the code.

Reply:
Thanks, I will hold this PR for now and will merge once Niki gives PR because tests builds are failing and if I merge this latest site will also fail and will not be accessible.